src/ORM/DataQueryManipulator.php

```diff
@@ -15,8 +15,14 @@ interface DataQueryManipulator
      * @param DataQuery $dataQuery
      * @param array $queriedColumns
      * @param SQLSelect $sqlSelect
+     * @param bool $preferConditions
      */
-    public function beforeGetFinalisedQuery(DataQuery $dataQuery, $queriedColumns = [], SQLSelect $sqlSelect);
+    public function beforeGetFinalisedQuery(
+        DataQuery $dataQuery,
+        $queriedColumns = [],
+        SQLSelect $sqlSelect,
+        $preferConditions = false
+    );
```

Comment:
Isn’t this a breaking API change? :(

Reply:
Yeah good point... I didn't consider that. I'll think about this a little more...

Reply:
I might move `preferConditions` to a property of the query. Then you can do something like `setPreferConditions(bool $preference);`.

Reply:
Resolving the conversation here but I'll link to it in the new PR I'll make soon

Reply:
I've decided to drop this idea. It's probably complexity that we don't need.